### PR TITLE
Clarifying migrate DB commands (BSC#1211729)

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/admin_saltcluster.xml [hunk 1]
```diff
@@ -1300,11 +1300,23 @@ devices                   /dev/vdb
         </step>
         <step>
           <para>
-            Migrate the DB device:
+            If the OSD does not have a pre-existing DB device, create new DB with the <command>new-db</command> command:
```
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1303]:
```suggestion
            If the OSD does not have a preexisting DB device, create a new DB with the <command>new-db</command> command:
```

File: xml/admin_saltcluster.xml [hunk 2]
```diff
@@ -1300,11 +1300,23 @@ devices                   /dev/vdb
...
+ --osd-fsid c276d2a4-5578-4847-94c6-8e2e6abf81c4 \
+ --target ceph-c3dc9227-ca3e-49bc-992c-00602cb3eec7/osd-db-b346b9ff-dbbe-40db-a95e-2419ccd31f2c</screen>
+          <para>
+            Then, migrate data using the <option>--from data</option> flag:
+<screen>[ceph: root@pacific /]ceph-volume lvm migrate --osd-id 0 \
+ --osd-fsid c276d2a4-5578-4847-94c6-8e2e6abf81c4 --from data \
+ --target ceph-c3dc9227-ca3e-49bc-992c-00602cb3eec7/osd-db-b346b9ff-dbbe-40db-a95e-2419ccd31f2c</screen>
+          </para>
+          <para>
+          If the OSD does have a pre-existing DB device, migrate the DB using the <option>--from db</option> flag:
```
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1315]:
```suggestion
          If the OSD has a preexisting DB device, migrate the DB using the <option>--from db</option> flag:
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see a couple of suggestions here. Thanks!

File: xml/admin_saltcluster.xml [hunk 3]
```diff
@@ -1300,7 +1300,19 @@ devices                   /dev/vdb
         </step>
         <step>
           <para>
-            Migrate the DB device:
+            If the OSD does not have a pre-existing DB device, create new DB with the <command>new-db</command> command:
```
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1303]:
```suggestion
            If the OSD does not have a preexisting DB device, create a new DB with the <command>new-db</command> command:
```

File: xml/admin_saltcluster.xml [hunk 4]
```diff
@@ -1300,7 +1300,19 @@ devices                   /dev/vdb
...
+ --osd-fsid c276d2a4-5578-4847-94c6-8e2e6abf81c4 \
+ --target ceph-c3dc9227-ca3e-49bc-992c-00602cb3eec7/osd-db-b346b9ff-dbbe-40db-a95e-2419ccd31f2c</screen>
+          <para>
+            Then, migrate data using the <option>--from data</option> flag:
+          </para>
+<screen>[ceph: root@pacific /]ceph-volume lvm migrate --osd-id 0 \
+ --osd-fsid c276d2a4-5578-4847-94c6-8e2e6abf81c4 --from data \
+ --target ceph-c3dc9227-ca3e-49bc-992c-00602cb3eec7/osd-db-b346b9ff-dbbe-40db-a95e-2419ccd31f2c</screen>
+          <para>
+          If the OSD does have a pre-existing DB device, migrate the DB using the <option>--from db</option> flag:
```
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1315]:
```suggestion
          If the OSD does have a preexisting DB device, migrate the DB using the <option>--from db</option> flag:
```

File: xml/admin_saltcluster.xml (hunk 3 above)
Comment (asettle) [reply xml/admin_saltcluster.xml:1303]: Huh, that's so weird it reverted - sorry @dariavladykina fixing again

**Pull request merged**